plugins/crust-gather.yaml

```diff
@@ -0,0 +1,48 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: crust-gather
+spec:
+  version: v0.5.3
+  homepage: https://github.com/crust-gather/crust-gather
+  platforms:
+  - bin: kubectl-crust-gather.exe
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_windows_amd64.tar.gz
+    sha256: 6c3e3697ad33ad190fd2398776fc9a2b28288147c57991878c9d48ed38063f23
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_linux_arm64.tar.gz
+    sha256: e4c01ab0b26af5771aba07b29ab303976841aace58b71605d816267f377d6df6
+    selector:
+      matchLabels:
+        os: linux
+        arch: arm64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_linux_amd64.tar.gz
+    sha256: 61b8dc87d4cabccc628fce55359b30d14b2d2363667d17696b98ba1f825f8582
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_darwin_arm64.tar.gz
+    sha256: 506fb5be53170cb2f9edb0842b616aed8e76c8b832f61d9e46ec2f636d880500
+    selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_darwin_amd64.tar.gz
+    sha256: 506fb5be53170cb2f9edb0842b616aed8e76c8b832f61d9e46ec2f636d880500
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  shortDescription: Provides functionality similar to os adm inspect, by collecting all resources for debugging purposes, and serving them via simple api-server.
```

Comment:
this won't fit and will be always truncated. limit to <60 chars.
https://krew.sigs.k8s.io/docs/developer-guide/plugin-manifest/

Reply:
Should be fixed.